Hipify contrib/nccl

xw285cornell opened the pull request:
Summary: hipify contrib/gloo

Test Plan: OSS & sandcastle build

Differential Revision: D18373308

bddppq commented:
Enable tests here https://github.com/pytorch/pytorch/blob/542d2f9ddf56b501b167ce105f55ee4a5ff1ac75/.jenkins/caffe2/test.sh#L84-L87

xw285cornell committed:
Hipify contrib/nccl (#29385)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/29385

hipify contrib/gloo

Test Plan: OSS & sandcastle build

Differential Revision: D18373308

fbshipit-source-id: 50ad3621d87a67b746cc415389b3d0278efdb2e9

xw285cornell commented:
nccl test seems to be passing :) https://ci.pytorch.org/jenkins/job/caffe2-builds/job/py2-devtoolset7-rocmrpm-centos7.5-test/44939/testReport/
